src/fusion.js

```diff
@@ -1,39 +1,31 @@
-import React, { createElement } from 'react'
-import { mapPropsStream } from 'recompose'
-import { Observable } from 'rxjs'
+import * as React from 'react'
+import { connect } from 'react-redux'
+import { default as compose } from 'recompose/compose'
+import { default as mapProps } from 'recompose/mapProps'
+import { mapPropsStreamWithConfig } from 'recompose/mapPropsStream'
+import { default as pure } from 'recompose/pure'
+import { default as rxjsConfig } from 'recompose/rxjsObservableConfig'
+import { ReplaySubject } from 'rxjs/ReplaySubject'
+import 'rxjs/add/operator/distinctUntilChanged'
+import 'rxjs/add/operator/publishReplay'
 
-export default propStream$ => (StreamedComponent) => {
-  class ComponentFromStream extends React.Component {
-    constructor(props, context) {
-      super(props, context);
-      const { store } = context;
-      if (!store) {
-        throw new Error(
-          `fuse() error: Missing Redux store in context.
-           Did you forget to <Provide> it?`
-        )
-      }
-      this.WrappedComponent = mapPropsStream(
-        propStream$(Observable.from(store), store.dispatch)
-      )(StreamedComponent)
-    }
+const mapPropsStream = mapPropsStreamWithConfig(rxjsConfig)
+const stateProxy = new ReplaySubject(1)
+const nullifiedState = {}
+const stateSpy = connect((storeState) => {
+  stateProxy.next(storeState)
+  return nullifiedState
+});
+const state$ = stateProxy.distinctUntilChanged().publishReplay(1).refCount()
+const dispatchSpy = compose(
+  stateSpy,
+  mapProps(({dispatch}) => dispatch),
```

Comment:
This should be `mapProps((dispatch) => dispatch)` since dispatch is passed as prop from connect if you don't specify `mapDispatchToProps` fn.  It then should be deconstructed from the `fuse` HOC